css/style.css

```diff
@@ -5,3 +5,33 @@ html {
 }
 
 /*put all other styles below this line, please*/
+
+ul {
```

Comment:
this selector is too broad... it will affect every unordered list on the page, which is probably not what you want to do.